src/serializers/index.ts

```diff
@@ -1,5 +1,6 @@
 import { Pipeline } from '../';
-
+import { Step } from 'src/base';
```

Comment:
```suggestion
import type { Step } from '../base';
```
I think?

Although, I think `Step` is exposed from `../`, sou you can probably just add a `, Step` to the existing import

Reply:
done